scipy/stats/_stats_py.py

```diff
@@ -8502,7 +8510,7 @@ def ks_1samp(x, cdf, args=(), alternative='two-sided', method='auto'):
     alternative = {'t': 'two-sided', 'g': 'greater', 'l': 'less'}.get(
         alternative.lower()[0], alternative)
     if alternative not in ['two-sided', 'greater', 'less']:
-        raise ValueError("Unexpected alternative %s" % alternative)
+        raise ValueError(f"Unexpected alternative: {alternative}")
```

Comment:
If you're going to add punctuation, might as well use your `alternative=` trick : )